homeassistant/components/bluetooth_le_tracker/device_tracker.py

```diff
@@ -80,6 +80,9 @@ def discover_ble_devices():
         except RuntimeError as error:
             _LOGGER.error("Error during Bluetooth LE scan: %s", error)
             return {}
+        except pygatt.exceptions.BLEError as error:
+            _LOGGER.error("BLEError during Bluetooth LE scan: %s", error)
+            return {}
```

Comment:
I would suggest to merge these two except statements into one since the errors are so similar and the type of error will be printed out as well.

```python
except (RuntimeError, pygatt.exceptions.BLEError) as error:
    _LOGGER.error("Error during Bluetooth LE scan: %s", error)
    return {}
```

Reply:
good idea, omw